packages/angular/src/polyfills.ts

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
What's the use of this file?

Reply:
It's required by Angular in order to support wide range of browsers. For instance, when I write ES6 JS, it would be converted to native JS, because some browsers doesn't support ES6. Full information can be found in the following docs - https://angular.io/guide/browser-support. Here you can read thorough explanation of the angular browser support and `polyfills.ts`.